tests/unit/test2_cpp/src/main.cpp

```diff
@@ -99,11 +100,25 @@ int main(void)
 	zcbor_multi_decode(1, 1, &one, (zcbor_decoder_t *)zcbor_int32_expect, state_d, (void*)14, 0);
 	zcbor_int32_expect(state_d, 15);
 	ret = zcbor_present_decode(&one, (zcbor_decoder_t *)zcbor_int32_expect, state_d, (void*)16);
-	
+
 	if (!ret) {
 		printk("Decode error: %d\r\n", zcbor_peek_error(state_d));
 		return 1;
 	}
+
+
+	struct Pet pet;
+	uint8_t input[] = {
+		0x83, 0x82, 0x63, 0x66, 0x6f, 0x6f, 0x63, 0x62, 0x61, 0x72,
+		0x48, 0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07, 0x08,
+		0x02};
+	int int_ret = cbor_decode_Pet(input, sizeof(input), &pet, NULL);
+
+	if (int_ret != ZCBOR_SUCCESS) {
+		printk("Decode error: %d\r\n", int_ret);
+		return 1;
+	}
+
```

Comment:
What does this have to do with the C++ support? Should we pull this out to a separate commit?

Reply:
It's a dummy test using C++, to test that the files can be #included from a .cpp file without problems.